docs/docs/20-usage/50-environment.md

```diff
@@ -63,7 +63,7 @@ This is the reference list of all environment variables available to your pipeli
 | `CI_COMMIT_SHA`                | commit sha                                                                                   |
 | `CI_COMMIT_REF`                | commit ref                                                                                   |
 | `CI_COMMIT_REFSPEC`            | commit ref spec                                                                              |
-| `CI_COMMIT_BRANCH`             | commit branch                                                                                |
+| `CI_COMMIT_BRANCH`             | commit branch (on pull request base branch)                                                  |
```

Comment:
```suggestion
| `CI_COMMIT_BRANCH`             | commit branch (equals source branch for pull requests)                                                  |
```

Reply:
no it dont! - this pull for example do have "master" as value

-> https://ci.woodpecker-ci.org/woodpecker-ci/woodpecker/build/1625/25

and more can be red in the code ;)

If this is the best way :/ dont thinks o

Reply:
So it is target branch? Purpose of my comment was just to re-use our existing wording, to not introduce new versions.

Reply:
It's target branch

Reply:
```suggestion
| `CI_COMMIT_BRANCH`             | commit branch (equals target branch for pull requests)                                                  |
```